src/assets/moneyStack.vue

```diff
@@ -0,0 +1,6 @@
+<template>
+    <svg width="57" height="50" viewBox="0 0 112 100" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
The SVG can be reduced:

`<svg xmlns="http://www.w3.org/2000/svg" width="57" height="50" fill="none" viewBox="0 0 112 100"><path fill="#04103B" d="M106.778 50c2.765-3.042 4.309-7 4.333-11.111a16.669 16.669 0 0 0-16.666-16.667H65.667A16.666 16.666 0 0 0 50 0H16.667A16.667 16.667 0 0 0 0 16.667a16.667 16.667 0 0 0 4.333 11.11 16.667 16.667 0 0 0 0 22.223 16.667 16.667 0 0 0 0 22.222A16.667 16.667 0 0 0 0 83.333 16.667 16.667 0 0 0 16.667 100h77.778a16.665 16.665 0 0 0 12.333-27.778 16.667 16.667 0 0 0 0-22.222ZM50 88.889H16.667a5.555 5.555 0 1 1 0-11.111H50a5.556 5.556 0 0 1 0 11.11Zm0-22.222H16.667a5.556 5.556 0 0 1 0-11.111H50a5.556 5.556 0 0 1 0 11.11Zm0-22.223H16.667a5.556 5.556 0 1 1 0-11.11H50a5.556 5.556 0 0 1 0 11.11Zm0-22.222H16.667a5.555 5.555 0 1 1 0-11.11H50a5.556 5.556 0 0 1 0 11.11Zm48.278 65.056a5.168 5.168 0 0 1-3.834 1.61H65.668a15.945 15.945 0 0 0 0-11.11h28.778A5.555 5.555 0 0 1 100 83.333a5.556 5.556 0 0 1-1.722 3.945Zm0-22.223a5.167 5.167 0 0 1-3.834 1.612H65.668a15.945 15.945 0 0 0 0-11.111h28.778A5.556 5.556 0 0 1 100 61.11a5.556 5.556 0 0 1-1.722 3.944Zm0-22.222a5.165 5.165 0 0 1-3.834 1.611H65.668a15.945 15.945 0 0 0 0-11.11h28.778A5.555 5.555 0 0 1 100 38.888a5.556 5.556 0 0 1-1.722 3.944Z"/></svg>`